lib/wanda/messaging/adapters/amqp/consumer.ex

```diff
@@ -25,10 +25,10 @@ defmodule Wanda.Messaging.Adapters.AMQP.Consumer do
 
   @impl GenRMQ.Consumer
   def handle_message(%GenRMQ.Message{payload: payload} = message) do
-    case Contracts.from_event(payload) do
-      {:ok, event} ->
-        Policy.handle_event(event)
-
+    with {:ok, event} <- Contracts.from_event(payload),
```

Comment:
Some doubts with this `with` XD

Reply:
that's ok, someone does not like using else with `with` blocks but I find it too strict

Reply:
`someone` is me but I think it's ok, no big deal

Reply:
it depends. if you have several `else` most probably you are better using other constructs, here we have a default error case that it does not just return but needs to call an error handling function